build.gradle

```diff
@@ -121,6 +121,16 @@ task renderEN(type: RenderCurriculumTask,
     }
 }
 
+// Executed by: ./gradlew -q recreateTermStructures
+tasks.register("recreateTermStructures") {
+    doLast {
+        exec {
```

Comment:
Can't we do this with a fine groovy script, like the others? Then we can just add the task to the renderDE job as prerequisite and include its definition. 

I'd like to take the idea as a starting point but would not like to start merging execution of bash scripts.